php/class-plugin.php

```diff
@@ -163,6 +169,9 @@ function init() {
 		if ( $this->is_module_active( 'deferred_customize_widgets' ) ) {
 			$this->deferred_customize_widgets = new Deferred_Customize_Widgets( $this );
 		}
+		if ( $this->is_module_active( 'customize_settings_snapshot' ) ) {
+			$this->customize_settings_snapshot = new Customize_Settings_Snapshot( $this );
```

Comment:
The Transactions patch directly changes `WP_Customize_Manager`, which we are not able to do here and cannot be extended as it's a `final` class. I'm all for isolating the functionality though the approach the Transaction patch took is not exactly possible. I guess I'm not completely following how we could make a `Customize_Snapshot_Manager` work like the patch does, but I'm happy to take another look and pivot on the PR. In your mind what would the responsibility of the `Customize_Snapshot_Manager` class be?